### PR TITLE
support node 12.x

### PR DESCRIPTION
**Pull request by sam-github**
Adds support for node 12.x.

Note: this doesn't break support for anything, I tested all the .travis.yml platforms back to 0.10, and it still works, so this can be published as semver-minor (adds 12.x support, a new feature).

### CODE REVIEW

**Comment by sam-github** [comment]
I tested with sucess on all the platforms, I'm curious to see what travis thinks:

```
git clean -x -d; nvm i 0.10; npm i && npm test; node --version
git clean -x -d; nvm i 0.12; npm i && npm test; node --version
git clean -x -d; nvm i 4; npm i && npm test; node --version
git clean -x -d; nvm i 5; npm i && npm test; node --version
git clean -x -d; nvm i 6; npm i && npm test; node --version
git clean -x -d; nvm i 8; npm i && npm test; node --version
git clean -x -d; nvm i 10; npm i && npm test; node --version
git clean -x -d; nvm i 11; npm i && npm test; node --version
git clean -x -d; nvm i 12; npm i && npm test; node --version
```

File: core.cc [hunk 1]
```diff
@@ -44,18 +44,26 @@ static char ident[1024];
...
+    int option = arg1->Value();
+    int facility = arg2->Value();
 
     openlog(ident, option, facility);
 
     return;
 }
 
-static char* dupBuf(const Handle<Value>& arg) {
+static char* dupBuf(const Local<Value>& arg) {
```
Comment (sam-github) [inline core.cc]: @bnoordhuis this surprised me, perhaps Local used to derive from Handle, and now it doesn't?

**Comment by sam-github** [comment]
@msimerson

**Comment by msimerson** [comment]
fixes #34

File: core.cc (hunk 1 above)
Comment (bnoordhuis) [reply core.cc]: That's right.

**Pull request merged**